creator-node/src/RehydrateIpfsQueue.js

```diff
@@ -1,14 +1,13 @@
 const Bull = require('bull')
 const config = require('./config')
-const { rehydrateIpfsFromFsIfNecessary, rehydrateIpfsDirFromFsIfNecessary } = require('./utils')
 const { logger: genericLogger } = require('./logging')
 
 const PROCESS_NAMES = Object.freeze({
   rehydrate_dir: 'rehydrate_dir',
-  rehydrate_file: 'rehydrate_file'
+  rehydrate_file: 'rehydrate_file',
```

Comment:
the separation of concerns between this file and rehydrateIpfsTasks.js is confusing

Reply:
for the bull queue to run separate processes, you must define each task in a separate file and destruct the job details per task in the separate file. 
https://github.com/OptimalBits/bull#separate-processes

so, `RehydrateIpfsQueue` adds specific tasks to the queue with the necessary job details. `rehydrateIpfsTasks.js` defines the logic per specific task (albeit this file could be renamed if this is where the confusion stems from). 

this is similar to how `ImageProcessingQueue` works too. that file adds specific tasks and its job details to the queue. then, `resizeImage.js` handles the task. 
https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/ImageProcessingQueue.js
https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/resizeImage.js

Reply:
let's add comments with this information, i also find this whole structure very confusing